provider/marathon/marathon_test.go

```diff
@@ -257,6 +257,124 @@ func TestMarathonLoadConfigNonAPIErrors(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "multiple ports",
```

Comment:
What's the "more part"? Maybe we can keep a more minimalistic test case in this test function but move the majority over to the `TestMarathonGetPort`.